scripts/dvm

```diff
@@ -50,6 +51,22 @@ _dvm_doctor() {
     echo "  RedHat/Fedora: sudo dnf install unzip"
     echo "  OpenSUSE:      sudo zypper install unzip"
     echo "  Arch/Manjaro:  sudo pacman -S unzip"
+    echo "  Mac OS:        brew install unzip"
+    return 1
+  fi
+}
+
+_dvm_check_fzf() {
+  if [[ ! -x "$(which fzf)" ]]; then
+    echo "ERROR: dvm requires the 'fzf' tool, which was not found."
+    echo "To install fzf:"
+    echo "  Debian/Ubuntu: sudo apt install fzf"
+    echo "  RedHat/Fedora: sudo dnf install fzf"
+    echo "  OpenSUSE:      sudo zypper install fzf"
+    echo "  Arch/Manjaro:  sudo pacman -S fzf"
+    echo "  Mac OS:        brew install fzf"
+    echo ""
+    echo "See: https://github.com/junegunn/fzf#installation"
```

Comment:
I think it'd be nice to use `fzf` if it's installed, but not require it. I actually hadn't heard of it till today, but it looks pretty nice!

Reply:
We only call `_dvm_check_fzf` if we are about to use `fzf`.

So if you do: `dart use <version>` you'll never notice... If you do `dart use` it'll error saying that you need `fzf`.

I did consider improving the error message to say that `dart use <version>` works without `fzf`, but looking at how widely `fzf` is available I think most users will just install it.